pkg/kubeconfig/create.go

```diff
@@ -10,9 +10,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes"
 	clientcmdlatest "k8s.io/client-go/tools/clientcmd/api/latest"
-	"k8s.io/klog"
-	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
+	"k8s.io/klog/v2"
 
+	kubeconfigutil "github.com/liqotech/liqo/internal/kubernetes/kubeconfig"
```

Comment:
@aleoli @palexster

Is the `CreateKubeConfigFromServiceAccount` function below still used after #802? It seems to me it is no longer referenced. In that case, it could be dropped, and also the backport of this package could be removed.

Reply:
I don't think it is used anymore, but waiting for @aleoli for confirmation.

Reply:
we can drop it